docs/DevGuide/Databox.md

```diff
@@ -374,7 +379,7 @@ NaiveAction<IntendedMutation>::apply(refined_databox);
 ~~~
 
 SpECTRE algorithms are decomposed into Actions which can depend on many more
-metavariables than we considered here. Feel free to look at the existing
-\ref ActionsGroup "actions that have been written." The intricacies of Actions
-at the level that SpECTRE uses them is the subject of a future addition to the
-\ref dev_guide "Developer's Guide."
+metavariables than ones we have considered here. Feel free to look at the
```

Comment:
I think I would remove the `ones` and also replace `many` with `a lot`. That would sound more natural to me.
I am not a native English speaker myself though.